lib/MilkCheck/Engine/BaseEntity.py

```diff
@@ -693,7 +693,8 @@ def inherits_from(self, entity):
         if self.target is None:
             self.target = entity.target
         self.mode = self.mode or entity.mode
-        self.remote = self.remote and entity.remote
+        if self.remote is None:
```

Comment:
This is not possible. Default remote value is True. So it will never be None. This test is always false.

Could you add a test to show the problem you're trying to solve?